### PR TITLE
Expose service account variable

### PR DESCRIPTION
**Pull request by cloud-pharaoh**
Default service account will not necessarily have the required permissions to run kubectl commands. Expose service_account_key_file variable from kubectl_wrapper module to use with ASM module.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by bharathkkb** [comment]
@cloud-pharaoh thanks for the PR!
LGTM, our lint seems to be failing on main so please rebase once #659 is merged

**Comment by cloud-pharaoh** [comment]
@bharathkkb I rebased and it's failing in a different spot now. Can you take a look at the failure?

**Comment by bharathkkb** [comment]
looks like that was flaky, we are tracking that here: https://github.com/hashicorp/terraform-provider-google/issues/6021

**Review by bharathkkb — APPROVED** [review]
Looks like inconsistent values are being returned for cloudrun config and that is failing one test. This has happened before and I have reopened b/159766801. ASM tests have passed so this LGTM
/cc @morgante

**Review by morgante — APPROVED** [review]

**Pull request merged**
